src/classes/BDI_FieldLevelSecurityService.cls

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
I was thinking about this over the weekend and I think I'm leaning towards leaving this interface out of the package for now, and just having BDI_DataImportFLSService exist as a standalone.

It's not a super strong opinion and it's maybe subjective so I could flex, but I'm just not I can see the future clearly enough to see when we need to share this behavior. We can try to schedule a quick huddle with some of the other devs as well, yeah - I can be flexible, but my preference would be to avoid providing the interface until we know more about where else this behavior would see use.